src/Cake.Common.Tests/Unit/Tools/GitVersion/GitVersionRunnerTests.cs

```diff
@@ -356,11 +356,11 @@ public void Should_Tolerate_Bad_Json_Set()
             }
 
             [Theory]
-            [InlineData(GitVersionVerbosity.None, nameof(Verbosity.Quiet))]
-            [InlineData(GitVersionVerbosity.Debug, nameof(Verbosity.Diagnostic))]
-            [InlineData(GitVersionVerbosity.Info, nameof(Verbosity.Verbose))]
-            [InlineData(GitVersionVerbosity.Warn, nameof(Verbosity.Normal))]
-            [InlineData(GitVersionVerbosity.Error, nameof(Verbosity.Minimal))]
+            [InlineData(GitVersionVerbosity.Quiet, nameof(Verbosity.Quiet))]
+            [InlineData(GitVersionVerbosity.Diagnostic, nameof(Verbosity.Diagnostic))]
+            [InlineData(GitVersionVerbosity.Verbose, nameof(Verbosity.Verbose))]
+            [InlineData(GitVersionVerbosity.Normal, nameof(Verbosity.Normal))]
+            [InlineData(GitVersionVerbosity.Minimal, nameof(Verbosity.Minimal))]
```

Comment:
These `InlineData` items should now use the new names int `GitVersionVerbosity`, e.g. 

```suggestion
            [InlineData(GitVersionVerbosity.Minimal, nameof(GitVersionVerbosity.Minimal))]
```

Reply:
Apologies, I figured you still wanted to verify that it mapped to the Cake verbosity level. Updating now.

Reply:
No worries! This particular test is checking if we are writing out the correct string when executing the tool. e.g.

```powershell
gitversion.exe -verbosity Minimal
```

So there's no mapping from Cake to GitVersion here. It's all GitVersion. There are other tests in the same file that do perform a mapping, for example if the user does not specify the verbosity they want for GitVersion - then we use the Verbosity of Cake to map to the best option for GitVersion.

Reply:
That makes sense to me - question then in this case:
If you no longer care to have the Cake Verbosity level, it would be better to remove the `nameof(Verbosity.x)` and just do the name of in the assert, no?

Reply:
i.e.
```csharp
[Theory]
[InlineData(GitVersionVerbosity.Quiet)]
[InlineData(GitVersionVerbosity.Diagnostic)]
[InlineData(GitVersionVerbosity.Verbose)]
[InlineData(GitVersionVerbosity.Normal)]
[InlineData(GitVersionVerbosity.Minimal)]
public void Should_Add_Verbosity_To_Arguments_If_Set(GitVersionVerbosity verbosity)
{
    // Given
    var fixture = new GitVersionRunnerFixture();
    fixture.Settings.Verbosity = verbosity;

    // When
    var result = fixture.Run();

    // Then
    Assert.Equal($"-verbosity {nameof(verbosity)}", result.Args);
}
```

Reply:
I don't see how you'd do the `nameof` in the assert, as `nameof` is resolved at compile time.

Reply:
You're most definitely right - I was meaning do a ToString() in the assert, just got it confused.

Something like:
```csharp
[Theory]
[InlineData(GitVersionVerbosity.Quiet)]
[InlineData(GitVersionVerbosity.Diagnostic)]
[InlineData(GitVersionVerbosity.Verbose)]
[InlineData(GitVersionVerbosity.Normal)]
[InlineData(GitVersionVerbosity.Minimal)]
public void Should_Add_Verbosity_To_Arguments_If_Set(GitVersionVerbosity verbosity)
{
    // Given
    var fixture = new GitVersionRunnerFixture();
    fixture.Settings.Verbosity = verbosity;

    // When
    var result = fixture.Run();

    // Then
    Assert.Equal($"-verbosity {verbosity.ToString()}", result.Args);
}
```
Compiles, passes, and prevents accidentally comparing GitVersionVerbosity to other Verbosity enums.

Reply:
I pushed a commit with the changes as you requested, just food for thought on testing that the enums are being added correctly to Args. I can adjust if you decide you like the ToString() method.

Reply:
Thanks! LGTM as-is.

Nothing particularly _wrong_ with the `ToString` approach, but for the sake of consistency with the other tests, the `nameof` seems slightly better to me given that the tests in this file (and many others in Cake codebase) follow this pattern of `TestName(input, expected)`.

It's also unlikely that this would ever happen, but if GitVersion decides in the future to add an option that uses a special character, or for example two words, say, `-verbosity "info only"`, then the `ToString` wouldn't match given that C# doesn't allow spaces, so the enum would have be `InfoOnly` or `Info_Only`. If we leave the `expected` open for any kind of string input, we could still easily test:

```csharp
// ...
[InlineData(GitVersionVerbosity.Minimal, nameof(GitVersionVerbosity.Minimal))]
[InlineData(GitVersionVerbosity.InfoOnly, "info only")]
```

Reply:
Good reasoning. Thanks for the feedback on the work.

Looking forward to helping out more!